src/documents/ModelRelay.md

```diff
@@ -129,10 +180,42 @@ that.child.applier.change("volume", 1);
 console.log(that.model.volume);       // 100 - inverse transformed to accept update from child component
 ```
 
-In general those transformations which are _**invertible**_ are the best choice for this kind of linkage. If the transforms are not invertible (or have no inverse defined in the framework), the updates will propagate only in one direction. This can still be highly useful.
+In general those transformations which are _**invertible**_ are the best choice for this kind of linkage. If the transforms are not invertible (or have no inverse defined in the framework), 
+the updates will propagate only in one direction. This can still be highly useful. In addition to its invertibility, the propagation of updates through a relay rule can be fine-tuned by the options
+`forward` and `backward`:
```

Comment:
I don't think we should have a colon at the end of a section like this. Perhaps instead the phrase ", described below" or ", described in the next section".